know/rdf/parser/sparql_reader.py

```diff
@@ -0,0 +1,18 @@
+import sys
+import os
+
+
+def read(query_file, uri):
```

Comment:
```suggestion
def read(query_file: str, uri: str) -> str:
```

types don't actually matter since they're just for the devs, but it's nice to have